jcapi-systemusers.go

```diff
@@ -93,13 +100,19 @@ func getJCUserFieldsFromInterface(fields map[string]interface{}, user *JCUser) {
 		user.ExternalSourceType = fields["external_source_type"].(string)
 	}
 
-	// Currently returned as int, not string (though they are posted as string),
+	// Currently returned as float64, not string (though they are posted as string),
 	// defect #96322248...
 	if _, exists := fields["unix_uid"]; exists {
-		user.Uid = getStringOrNil(fields["unix_uid"])
+		floatVal, ok := fields["unix_uid"].(float64)
```

Comment:
purely cosmetic, but makes for tighter code:

if floatVal, ok := fields["unix_uid"].(float64); ok {
			user.Uid = strconv.FormatInt(int64(floatVal), 10)
		}